plinkSTR/README.md

```diff
@@ -0,0 +1,61 @@
+# PlinkSTR 
```

Comment:
let's hold off on PlinkSTR readme for now. make a local copy of your edits, but change back to "Under construction" until we talk about this one further